content/06.discussion-a.md

```diff
@@ -1,82 +1,69 @@
-Despite the many potential applications of GitHub to EEB research, we acknowledge that there will still be many times when researchers might look to other platforms for research collaboration. 
-
-### When to look to other platforms for collaboration  
-*Contributors to this section: Rob*  
-Though we see GitHub as a useful tool for collaboration in EEB, we describe 2 use cases where, to our knowledge, GitHub's features still fall short of the type of highly collaborative work emblamatic of EEB research.  
-First, because of the underlying version control system which tracks "pushed" changes through "commits", real-time collaborative editing (e.g., as on a shared Google Doc or a Word document stored on Dropbox) is not possible on GitHub.  
-There are now websites outside of the GitHub ecosystem that are built on top of the GitHub architecture that allow real-time collaboartive editing (e.g., [hackMD](https://hackmd.io/).  
-We used HackMD at two key points in writing our manuscript when real-time co-writing was essential: when taking meeting notes and writing the outline of our paper.  
-Despite the extra layer of coordination required to use GitHub to write a paper, many journals are changing their manuscript formatting requirements to allow for manuscript format output from GitHub [@doi:10.1038/d41586-020-00916-6].  
-Second, we looked to other software when working on figures and tables.
-Creating tables and figures on GitHub using markdown or other scripting languages is possible, we found that it was not practical at the early brainstorming stages.
+Many of the examples are specifically meant for first-time GitHub users and can likely be adopted with just several hours of practice (e.g., storing data, creating virtual notebooks, making code citable).  
+For example, storing code and data and making it citable simply involves creating a repository on Github, pushing code to the repository, and then going through several quick steps to create a DOI for the repository.   
+Even with time necessary to learn some GitHub and version control basics, we emphasize that these are transferrable skills that can help students and postdocs be successful in academia or industry.  
+
+Other more advanced examples in this paper may require a greater time commitment, but have the potential to make EEB research even more open, accessible, and collaborative than ever before.  
+For example, managing full research projects or research labs on GitHub will require careful thought as to how to delegate tasks such as reviewing pull requests or creating issues.  
+Additionally, collaboratively writing a manuscript using GitHub, as we have done here, involves an entire team to be engaged in more advanced GitHub processes like using GitHub actions and reviewing pull requests.   
```

Comment:
```suggestion
Additionally, collaboratively writing a manuscript using GitHub, as we have done here, involves an entire team to be engaged in more advanced GitHub processes like making and reviewing pull requests.   
```
While GitHub actions was important, the *whole team* didn't need to know it.  It was helpful (essential even) to have someone (@pedrohbraga) on the team to spearhead the setup and maintenance of the more technical aspects of GitHub.